src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -1530,6 +1537,54 @@ public void shouldReturnErrorResponseIfAllInstancesFailed()
     assertThat(instances.size(), is(0));
   }
 
+  @Test
+  public void canCreateAnInstanceWithNatureOfContent() throws Exception {
```

Comment:
Could the existing test for creating an instance have included this new property instead of a new test?

Reply:
Hi @marcjohnson-kint, sure, no prob, will do so.

Reply:
Hi @marcjohnson-kint, fixed, thanks.